tests/virt/mod.rs

```diff
@@ -199,3 +222,37 @@ impl HidDevice for Device<'_> {
         }
     }
 }
+
+fn with_client<F, T>(files: &[(&Path, &[u8])], f: F) -> T
+where
+    F: FnOnce(Client<Ram>) -> T,
+{
+    virt::with_platform(Ram::default(), |mut platform| {
+        // virt always uses the same seed -- request some random bytes to reach a somewhat random
+        // state
+        let uniform = Uniform::from(0..64);
+        let n = uniform.sample(&mut rand::thread_rng());
+        for _ in 0..n {
+            platform.rng().next_u32();
+        }
```

Comment:
I do not understand why you virt random should be truly random? `virt` is made that way on purpose to make the tests reproducible.

And if we want to have true `rng` with virt, why not use:


```
*platform.rng() = ChaCha8Rng::from_rng(&mut thread_rng());
```

Reply:
> KEK and KWK actually worked or if we just re-generated the same keys

In that case I would prefer to have a second constant seed, rather than truly random, for the part of the test that needs distinct RNG.

Reply:
Let’s do that separately.  I’ve created an issue for it: https://github.com/Nitrokey/fido-authenticator/issues/88